testing/features/step_definitions/header_steps.rb

```diff
@@ -23,6 +23,7 @@
 end
 
 Then("I am able to search for {string}") do |search_term|
+  @component.open_search_pane.click if device?
```

Comment:
Might be worth naming this `mobile_phone?` or similar?

Strictly speaking this behaviour is tied to screen size rather than device ID.

Reply:
Good point, will rename. 

Also whilst we don't test on small resolutions, due to % compatibility e.t.c. it might be worth something revisiting later on. I don't think it's worth spending a lot of time on though.